### ISSUE TITLE
Add Cooper Hewitt font family

### ISSUE THREAD

**Issue by KyleAMathews** [google/fonts#322]
Fantastic font sponsored by the Smithsonian design museum of the same name. Would be a great addition to Google Fonts.

Licensed under the SIL Open Font License and can be downloaded at http://www.cooperhewitt.org/open-source-at-cooper-hewitt/cooper-hewitt-the-typeface-by-chester-jenkins/

A review of the font: http://typographica.org/typeface-reviews/cooper-hewitt-the-typeface/

**Comment by davelab6** [comment]
Its blocked on trademark permission

**Comment by impallari** [comment]
Since there is so little activity on the cooperhewitt-typeface github, what about forking + renaming?

**Comment by davelab6** [comment]
@impallari for me that is less than ideal, but it is a Plan B option :) I'll follow up with C-H on this soon

**Comment by kvendetti** [comment]
Hello, I was hoping to use the Cooper Hewitt font (it is part of our branding guidelines).  It wasn't clear to me if this is going to be possible or what the next best alternative would be?

**Comment by davelab6** [comment]
This is unlikely to be available from Google Fonts for a couple month, at least

**Comment by kvendetti** [comment]
bummer, thanks for the quick response.

Kelly Vendetti
Breakaway Prep, Utah
875 Iron Horse Drive
Park City, Utah 84060
(303) 547-6275

**Comment by sambaldwin** [comment]
+1

**Comment by piperhaywood** [comment]
+1

**Comment by silicondales** [comment]
Any movement on this at all?

**Comment by kvendetti** [comment]
i gave up and moved on with a different font.  So, I am not sure.

*Kelly Vendetti*
Breakaway Prep Park City
ACT/SAT Test Preparation, Tutoring, and Admissions Counseling
875 Iron Horse Drive B
Park City, Utah 84060
(435) 487-9651
www.breakaway-prep-park-city.com
www.facebook.com/breakawayutah

**Comment by robin-scott** [comment]
Yeah I think we went another way, but the license seems to be okay for this. Trademarks - as far as I know, permission from the trademark holder should be sufficient?! Its a common font choice owing to the license, so seems like it would be a good addition to G Fonts.

**Comment by alinsan** [comment]
+1  - would love to see this added as well

**Comment by thundernixon** [comment]
Upstream repo at https://github.com/cooperhewitt/cooperhewitt-typeface

**Comment by thundernixon** [comment]
> Hello, I was hoping to use the Cooper Hewitt font (it is part of our branding guidelines). It wasn't clear to me if this is going to be possible or what the next best alternative would be?

For what it's worth, if anyone else comes here with a similar question before this is up on the Google Fonts directory, it still _is_ 100% legal and safe to use the Cooper Hewitt font in commercial projects, due to [its OFL license](https://github.com/cooperhewitt/cooperhewitt-typeface/blob/master/files/-CooperHewitt-OFL-201406.txt). If you use it on a website, you'd just need to self-host the files (at least until it can be put onto the GF directory).

**Comment by KyleAMathews** [comment]
You can also self-host using this NPM package https://www.npmjs.com/package/typeface-cooper-hewitt

**Comment by whatsnewsisyphus** [comment]
+1 for seeing it hosted on google

**Comment by tasuki** [comment]
Hello from 2021!

**Comment by langlukas** [comment]
+1

**Comment by cosmojg** [comment]
@RosaWagner Why the "-- missing sources" tag?

Cooper Hewitt's upstream repo seems to provide sources. In case you missed it, it can be found here: https://github.com/cooperhewitt/cooperhewitt-typeface

**Comment by mgrn0** [comment]
+1

**Comment by Vladi-Dimov** [comment]
Still no progress about this font's copyright issue for like 9 (nine!) years now, author doesn't seems active anymore since at least 2019, so the issue will remain permanently UNRESOLVED.

**We may be better do: give up and discard this font entirely now** (as its adding being basically impossible).
